google/datalab/contrib/pipeline/composer/setup.sh

```diff
@@ -0,0 +1,25 @@
+ZONE=us-central1
+PROJECT=$1
+EMAIL=$2
+ZONE=$3
+ENVIRONMENT=$4
+
+
+gcloud config set project $PROJECT
+gcloud auth login --activate $EMAIL
+sudo gcloud components repositories add https://storage.googleapis.com/composer-trusted-tester/components-2.json
```

Comment:
First thing is to install composer specific gcloud and it may vary slightly depending on the environment (e.g., VM, shell, local) before any gcloud * operations.